src/components/tasks/Checkbox.js

```diff
@@ -17,7 +17,7 @@ const CheckboxStyled = styled.input.attrs({ type: 'checkbox' })`
   &:checked {
     background-image: url(/images/checkmark.png);
     background-position: center;
-    animation: pulse 1s;
+    animation: pulse 0.75s;
```

Comment:
by the way, thanks for having the animation, because that's why I looked at your code and then at mine and I was able to shorten it (it behaved difficult before, no idea why not anymore, but that's ok ;))